DevVehicle35-Motors/App/Program.cs

```diff
@@ -23,6 +25,9 @@
             case 3:
                 QuadInteraction.BuildQuad();
                 break;
+            case 4:
+                new MiniBus();
```

Comment:
Use an static method.

Reply:
we did a new interaccion class to minibuss, ty